src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -549,6 +551,103 @@ def evaluate(self, dataset, metric='auto'):
         """
         return self.__proxy__.evaluate(dataset, metric)
 
+    def predict_topk(self, dataset, output_type='probability', k=3, output_frequency='per_row'):
```

Comment:
This looks like an implementation of `predict_topk`, not a fix for a breakage. What was broken?

Reply:
The unit test test_predict_topk for AC was throwing en error : 
AttributeError: 'ActivityClassifier' object has no attribute 'predict_topk'.
Sl the fix is just to add the function to the beta class I guess.

Reply:
Same for the test_classify as well.